### PR TITLE
RSWEB-7668 feat(zoolander): added class for external links

### PR DESCRIPTION
**Pull request by runrog**
Added the class for the external link icon. Once brought in, we can add them to the needed links in Drupal.

### CODE REVIEW

**Comment by runrog** [comment]
Shots of nav and footer
screen shot 2016-11-14 at 11 34 26 am
screen shot 2016-11-14 at 11 35 11 am

**Comment by WidgetsBurritos** [comment]
👍

**Comment by runrog** [comment]
@WidgetsBurritos I'll add it to the pattern page in zoolander also. The link color should inherit from the closest link color rule.

**Comment by runrog** [comment]
@WidgetsBurritos Check out the new update. if you go to the http://localhost:9000/derek/examples/solutions
page, scroll to the bottom, just above your "Text" entry.

File: styleguide/_themes/derek/scss/globalElements/ext-links.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    font-family: 'FontAwesome';
+    margin-left: 5px;
+  }
+}
+
+.footer,
+.main-nav {
+  .extLink {
+
+    &::after {
```
Comment (bighappyface) [inline styleguide/_themes/derek/scss/globalElements/ext-links.scss]: How about another class for this instead of nested classes?

Comment (runrog) [inline styleguide/_themes/derek/scss/globalElements/ext-links.scss]: that works too @bighappyface

Comment (runrog) [inline styleguide/_themes/derek/scss/globalElements/ext-links.scss]: @bighappyface changes made

**Comment by WidgetsBurritos** [comment]
👍

**Comment by WidgetsBurritos** [comment]
Given that this is all new functionality, I'm gonna go ahead and merge.

**Pull request merged**